docparse/jsonschema.go

```diff
@@ -354,7 +368,7 @@ arrayStart:
 	// Simple identifier: "string", "myCustomType".
 	case *ast.Ident:
 
-		dbg("resolveArray: ident: %#v", typ.Name)
+		dbg("resolveArray: ident: %#v in %s", typ.Name, pkg)
```

Comment:
Very minor thing, but IMHO it's better to use `%#v` or `%q` in debug messages. That way things like trailing spaces, empty strings, etc. show up a bit clearer.